src/applications/vaos/tests/appointment-list/components/AppointmentsPage/index.unit.spec.jsx

```diff
@@ -93,10 +91,9 @@ describe('VAOS integration: appointment list', () => {
     });
 
     const { baseElement, findAllByRole, getByText } = renderWithStoreAndRouter(
-      <FutureAppointmentsList />,
+      <AppointmentsPage />,
```

Comment:
I had to switch a bunch of uses of FutureAppointmentsList to AppointmentsPage in unit tests because they were all running in the mode where the EC flag is off and the appoinment list service call is kicked off within the tab components. Now those fetches are only kicked off from AppointmentsPage.

This will flip back again with the homepage refresh.